netbox/templates/virtualization/virtualmachine.html

```diff
@@ -60,8 +60,8 @@ <h5 class="card-header">
                             <a href="{% url 'ipam:ipaddress' pk=object.primary_ip6.pk %}">{{ object.primary_ip6.address.ip }}</a>
                             {% if object.primary_ip6.nat_inside %}
                               (NAT for <a href="{{ object.primary_ip6.nat_inside.get_absolute_url }}">{{ object.primary_ip6.nat_inside.address.ip }}</a>)
-                            {% elif object.primary_ip6.nat_outside %}
-                              (NAT: <a href="{{ object.primary_ip6.nat_outside.get_absolute_url }}">{{ object.primary_ip6.nat_outside.address.ip }}</a>)
+                            {% elif object.primary_ip4.nat_outside.count > 0 %}
```

Comment:
Careful with the copy/pastes. :slightly_smiling_face: 

```suggestion
                            {% elif object.primary_ip6.nat_outside.exists %}
```